src/extension.ts

```diff
@@ -55,6 +55,53 @@ export function activate(context: vscode.ExtensionContext) {
 			console.error(e);
 		}
 	}));
+
+	// Generate a the CRUD workflow for the C# model based on the configurations
+	context.subscriptions.push(vscode.commands.registerCommand('csharp-bootstrapper.bootstrap-crud', (uri:vscode.Uri) => {
+		try{
+			if (uri) {	
+				// Get the document text
+				const documentText: string = fs.readFileSync(uri.path).toString();
+
+				const className: string = getClassName(documentText);
```

Comment:
`getClassName` has been updated to get multiple class names. I'm currently working on updating it into `parseClass` that will spit out a struct that we can pass around and use various components of (Which will also simplify using it so we don't have to keep using array access).

2 ideas:
1. Merge master into this branch and get it working
2. Wait for the finished class parser

Reply:
Master doesn't have the getClassName function's update.

Reply:
Ah, good point. I just made the PR for that. Would you rather merge this in and I can update that branch, or the other way around?